retrieval/series_cache.go

```diff
@@ -114,6 +115,15 @@ type seriesCacheEntry struct {
 
 	// Last time we attempted to populate meta information about the series.
 	lastRefresh time.Time
+
+	// Whether the series needs to be exported.
+	exported bool
```

Comment:
the variable is not clear. It can be 'it is exported', or 'need to be exported.'

Reply:
'need to be exported' and 'is being exported' are the same here, so it's both. I am happy to change the name if you suggest a specific alternative.